src/components/lib.js

```diff
@@ -0,0 +1,8 @@
+import CyverseAnnouncer, { announce } from "./announcer/CyverseAnnouncer";
+import Rate from "./rating/Rate";
+
+export {
+    CyverseAnnouncer,
+    announce,
+    Rate,
+};
```

Comment:
Any specific reason why this was renamed to `lib.js`?

Reply:
Not really. I was kind of following the recommendation of https://hackernoon.com/creating-a-library-of-react-components-using-create-react-app-without-ejecting-d182df690c6b and making `lib` as the main entry point of the library.

Since `src/index.js` was not necessary, I just renamed `src/components/index.js` to `src/components/lib.js`, but this export can be done from `src/lib.js` or `src/index.js` if we want. Just as long as we pass the correct path to `babel` in the package.json `build` script.